experimental/rel_component/configs/rel_trf.cfg

```diff
@@ -0,0 +1,119 @@
+[paths]
+train = null
+dev = null
+raw = null
+init_tok2vec = null
+
+[system]
+seed = 342
+gpu_allocator = "pytorch"
+
+[nlp]
+lang = "en"
+pipeline = ["transformer", "relation_extractor"]
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.transformer]
+factory = "transformer"
+max_batch_items = 4096
+set_extra_annotations = {"@annotation_setters":"spacy-transformers.null_annotation_setter.v1"}
+
+[components.transformer.model]
+@architectures = "spacy-transformers.TransformerModel.v1"
+name = "roberta-base"
+tokenizer_config = {"use_fast": true}
+
+[components.transformer.model.get_spans]
+@span_getters = "spacy-transformers.strided_spans.v1"
+window = 16
```

Comment:
That's a pretty small window. Is it necessary to make it so small to get it to train on your GPU? The stride of 4 also pretty odd.

With (16, 4) you'll have slices like this in a 40 word text: `(0, 16), (4, 20), (8, 24)` etc. So each word will actually be getting encoded 4 times, but each time you'll have a context window of 16. It's better to have something like `(0, 16), (12, 28), ...` etc. The stride still gives each word a chance to see some context on either side, which you wouldn't get if you had windows like `(0, 16), (16, 32), (32, 40)` --- in that approach, there's no window that has tokens 15 and 16 together. So it's good to have the stride lower than the window, but it's almost never going to be good to have the `window > stride * 2`.

Reply:
I hadn't tuned (or thought about) these parameters yet before - thanks for the explanation!